### PR TITLE
Create footer

### PR DESCRIPTION
**Pull request by m-chit**

### CODE REVIEW

**Comment by m-chit** [comment]
https://codeburst.io/4-four-ways-to-style-react-components-ac6f323da822 - 3. CSS Modules

**Comment by m-chit** [comment]
screenshot from 2018-07-27 13-41-42

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import React, { Component } from "react";
 import "./App.scss";
-import logo from './images/Logo.svg';
-
+import Footer from "./components/Footer/Footer";
```
Comment (kgrining) [inline src/App.js]: --- External imports ---

--- Internal imports ---

--- styles ---

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import styles from "./style.scss";
+import logo from "../../images/Logo.svg";
```
Comment (kgrining) [inline src/components/Footer/Footer.js]: same as in App.js

File: src/index.scss [hunk 3]
```diff
@@ -1,4 +1,5 @@
 body {
   margin: 0;
   padding: 0;
+  background-color: #ffdfdf;
```
Comment (kgrining) [inline src/index.scss]: create colors.scss and import $someColor

**Pull request merged**